cmd/scw/testdata/test-all-usage-config-validate-usage.golden

```diff
@@ -0,0 +1,15 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Validate the config
```

Comment:
What does it mean? Will it test for YAML correctness? Will it check that the config contains only valid fields? Please explain more about what this validation will mean.